### DISCUSSION TITLE
Como fazer o app sobrepor a outros

### DISCUSSION THREAD

**Discussion by emily-lar** [Flutterando/forum#162] [Perguntas e Respostas]
Olá, 
Estou precisando fazer a tela de aceite do app sobrepor aos outros aplicativos que o usuário esteja usando porém não sei se existe algum plugin ou uma forma mais rápida. 

Basicamente o aplicativo necessita que o usuário dê um aceite quando deseja aceitar uma nova solicitação, porém muitas vezes esse usuário está com a tela bloqueada ou usando outro app e precisa abrir o meu app pra conseguir aceitar essa solicitação. 

Gostaria de saber se tem como criar uma tela como se fosse um widget pra que ele possa aceitar e que esse "widget" se sobreponha a outros apps. 

Obrigada pela ajuda.

**Comment by claupereirajr** [comment]
Olha Emily pelo que lembro, não existe tal função, forçar uma sobreposição do seu app para outro. Acho que a forma mais fácil/ correta seria um push notification, e tentar fazer algo do tipo a notificação do "Whatsapp" que quando você carrega aparece as opções de "Responder", "Marcar como lido".

**Reply by emily-lar** [reply]
No caso não existe nenhum plugin que faça esse tipo de notificação?

**Comment by rajada1** [comment]
> 

Eu fico com a resposta do @claupereirajr , mas vou deixar aqui um package que vc pode usar para sobrepor outros apps do sistema. 
https://pub.dev/packages/flutter_overlay_window

**Reply by emily-lar** [reply]
Obrigadaaa.
